packages/backend/src/server/api/ApiServerService.ts

```diff
@@ -126,17 +127,18 @@ export class ApiServerService {
 				username: string;
 				password: string;
```

Comment:
#14679 で変えそびれてしまったので一応
```suggestion
				password?: string;
```

Reply:
#14705